ci/docker/python-wheel-musllinux.dockerfile

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG base
+FROM ${base}
+
+ARG arch
+ARG arch_short
+ARG musllinux
+
+ENV LINUX_WHEEL_NAME='musl'
+ENV LINUX_WHEEL_VERSION=${musllinux}
+
+RUN apk update
+RUN apk add --no-cache build-base cmake flex wget zip ninja ccache
+
+# A system Python is required for ninja and vcpkg in this Dockerfile.
+# On manylinux2014 base images, system Python is 2.7.5, while
+# on manylinux_2_28, no system python is installed.
+# We therefore override the PATH with Python 3.8 in /opt/python
+# so that we have a consistent Python version across base images.
```

Comment:
Could you update this comment?
(Do we need this workaround for musllinux too?)

Reply:
On musl it has python installed (3.12) but there is no pip installed. I can either install it via `apk add py3-pip` or I can set `PATH` to `/opt/python/cp12-cp12/bin`. Maybe setting the `PATH` would be better as it doesn't need to install anything else.